blog/docs/releases/announcing-knative-v1-17-release.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "v1.17 release"
+linkTitle: "v1.17 release"
+author: "[ [Dave Protasowski (independent)](https://github.com/dprotaso), [David Simansky (Red Hat)](https://github.com/dsimansk) and [Stavros Kontopoulos (Red Hat)](https://github.com/skonto)"
+author handle: https://github.com/dprotaso https://github.com/dsimansk https://github.com/skonto
+
+date: 2025-01-28
+description: "Knative v1.17 release announcement"
+type: "blog"
+---
+
+# Announcing Knative 1.17 Release
+
+![Release 1.17](./images/release1.17-1.png)
+A new version of Knative is now available across multiple components. Follow the instructions in [Installing Knative](https://knative.dev/docs/install/) to install the components you require.
+
+This release brings a number of smaller improvements to the core Knative Serving and Eventing components, and several improvements to specific plugins.
+
+## Table of Contents
+- [Serving](#serving)
+- [Eventing](#eventing)
+- [kn CLI](#kn-cli)
+- [Functions](#functions)
+- [Knative Operator](#knative-operator)
+
+### Serving
+**Release Notes:** [Knative Serving 1.17](https://github.com/knative/serving/releases/tag/knative-v1.17.0)
+
+#### 💫 New Features & Changes
+- Adding support for the "hostPath" volume type. This feature is behind the flag `kubernetes.podspec-volumes-hostpath`. ([#15648](https://github.com/knative/serving/pull/15648), @amarflybot)
+- A revision is now set to ContainerHealthy=True when it has ready replicas ([#15503](https://github.com/knative/serving/pull/15503), @SaschaSchwarze0)
```

Comment:
This is a bug, the bug label was missing. I added it. Want to move it to the bugs section?